sass/_components.scss

```diff
@@ -275,11 +275,13 @@
             margin-bottom: $baseline;
 
             @include mobile {
+                font-size: $size-5;
```

Comment:
You can has the class "is-size-5-mobile" on elements to scale text per breakpoint :) See http://bulma.io/documentation/modifiers/typography-helpers/